src/lib.rs

```diff
@@ -333,6 +334,106 @@ pub struct ApiError {
     pub error_description: Option<String>,
 }
 
+/// # Example
+///
+/// ```
+/// # extern crate mammut;
+/// # use mammut::StatusesRequest;
+/// let request = StatusesRequest::default()
+///                               .only_media()
+///                               .pinned()
+///                               .since_id("foo");
+/// # assert_eq!(&request.to_querystring()[..], "?only_media=1&pinned=1&since_id=foo");
+/// ```
+#[derive(Clone, Debug)]
+pub struct StatusesRequest {
+    only_media: bool,
+    exclude_replies: bool,
+    pinned: bool,
+    max_id: Option<String>,
+    since_id: Option<String>,
```

Comment:
Both `max_id` and `since_id` should be `Cow<str>` we don't require ownership of the strings.